specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/mfe.json

```diff
@@ -0,0 +1,1719 @@
+        },
+        "glusterFs": {
+          "$ref": "#/definitions/GlusterFsSection"
+        }
+      }
+    },
+    "OriginType": {
+      "enum": [
+        "Synapse"
+      ],
+      "type": "string"
+    },
+    "LinkedInfo": {
+      "type": "object",
+      "properties": {
+        "linkedId": {
+          "type": "string"
+        },
+        "linkedResourceName": {
+          "type": "string"
+        },
+        "origin": {
+          "$ref": "#/definitions/OriginType"
+        }
+      }
+    },
+    "DatastoreProperties": {
+      "required": [
+        "contents"
+      ],
+      "type": "object",
+      "properties": {
+        "contents": {
+          "$ref": "#/definitions/DatastoreContents"
+        },
+        "description": {
+          "type": "string"
+        },
+        "hasBeenValidated": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "id": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true
+        },
+        "isDefault": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "linkedInfo": {
+          "$ref": "#/definitions/LinkedInfo"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "DatastorePropertiesResource": {
+      "description": "Azure Resource Manager resource Envelope",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DatastoreProperties"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "DatastorePropertiesResourcePaginatedResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DatastorePropertiesResource"
+          }
+        },
+        "continuationToken": {
+          "type": "string"
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "AssetPath": {
+      "description": "Details of an AssetUri.",
+      "type": "object",
+      "properties": {
+        "path": {
+          "description": "The path of file/directory.",
+          "type": "string"
+        },
+        "isDirectory": {
+          "description": "Whether the path defines a directory or a single file.",
+          "type": "boolean"
+        }
+      }
+    },
+    "ModelVersion": {
+      "description": "Details of an ModelDto",
+      "type": "object",
+      "properties": {
+        "stage": {
+          "description": "Model asset stage.",
+          "type": "string"
+        },
+        "name": {
```

Comment:
Same comment as modelcontainer, "name" should be renamed here since "name" is already a top level property